npx tsdx create fails with 0.10

### Current Behavior

`npx tsdx create mylib` fails (before template prompt)

### Expected behavior

`npx tsdx create mylib` works

### Suggested solution(s)

`npx tsdx@0.9 create mylib`

### Your environment

<!-- PLEASE FILL THIS OUT -->

| Software         | Version(s) |
| ---------------- | ---------- |
| TSDX           | 0.10
| TypeScript       | not installed globally
| npm/Yarn         | yarn 1.19, npm 6.12
| Operating System | macOS

I think I've having the same issue.  Easy way to reproduce, for me, was to run:

```
$ docker run -it node
# npx tsdx create mylib
```

But I also get the same error running `npx tsdx create mylib` under Windows (`Cannot find module 'typescript'`).

If, instead, I run `yarn add tsdx` so that it is installed locally in a package I get the error message: `warning " > tsdx@0.10.0" has unmet peer dependency "typescript@3".`.  The result is that if, from that repo where `tsdx` is installed locally I do `npx tsdx create ../foo`, I again get `Cannot find module 'typescript'`.

However, doing "yarn add typescript tsdx" and then running `npx tsdx create ../foo` it then runs.

Bottom line, it seems that having `typescript` as a peer dependency is not a good idea for something to be run via `npx`.  I suspect it was globally installed during testing or something which is how this was missed.

Same error here:

```bash
$ npx tsdx create tsdx-demo
Cannot find module 'typescript'
```

Software | Version(s)
------------ | -------------
TSDX | 0.10
TypeScript | not installed globally
Yarn | 1.19.1
npm | 6.9.0
node | 10.16.3
OS | Ubuntu 18.04

Using `npx tsdx@0.9 create tsdx-demo` worked.

--- a/package.json
+++ b/package.json
@@ -115,9 +115,6 @@
     "shelljs": "^0.8.3",
     "typescript": "^3.6.2"
   },
-  "peerDependencies": {
-    "typescript": "3"
-  },
   "husky": {
     "hooks": {
       "pre-commit": "pretty-quick --staged --pattern '!test/tests/lint/**' && yarn lint"

